.github/bin/label_abandoned_prs.py

```diff
@@ -46,5 +46,5 @@
         subprocess.run(['gh', 'pr', 'comment', num, '--body', inactive_comment])
     if ((tdiff > two_weeks and not already_abandoned) or (tdiff > two_days and already_warned)) and not approved:
         subprocess.run(['gh', 'pr', 'edit', num, '--add-label', 'Abandoned PR - Needs New Owner'])
-    if approved:
+    if approved and not already_abandoned:
```

Comment:
Is this what you want? This next subprocess is to remove the abandoned PR label, so shouldn't the PR be already abandoned? (since if it isn't abandoned, there is no label to remove)

Reply:
Also, if it's not approved, nor has requested changes, the 'approved' variable will not be defined, this is one reason why the action was failing earlier.

Reply:
Ah, that's a typo, let me fix that.